server/server.js

```diff
@@ -0,0 +1,110 @@
+const express = require("express");
+const cors = require("cors");
+const pool = require("./database");
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+
+app.post("/addUser", (req, res) => {
```

Comment:
Similar to the above https://github.com/Stacktrek-Training/stack-ilearning/pull/28/files#r1196092317

```suggestion
app.post("/users", (req, res) => {
```

Reply:
Though not sure if we actually need to save users in the DB. If the goal is to use the users of stacktrek.com, then this wouldn't be needed anymore.